Counterexamples/CliffordAlgebra_not_injective.lean

```diff
@@ -0,0 +1,273 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.Algebra.CharP.Algebra
+import Mathlib.Algebra.CharP.Pi
+import Mathlib.Algebra.CharP.Quotient
+import Mathlib.Algebra.CharP.Two
+import Mathlib.Data.Matrix.Notation
+import Mathlib.Data.MvPolynomial.CommRing
+import Mathlib.Data.Nat.Prime
+import Mathlib.Data.ZMod.Basic
+import Mathlib.FieldTheory.MvPolynomial
+import Mathlib.LinearAlgebra.CliffordAlgebra.Basic
+import Mathlib.LinearAlgebra.Finsupp
+import Mathlib.RingTheory.MvPolynomial.Basic
+import Mathlib.RingTheory.MvPolynomial.Ideal
+
+/-! # `algebraMap R (CliffordAlgebra Q)` is not always injective.
+
+A formalization of https://mathoverflow.net/questions/60596/clifford-pbw-theorem-for-quadratic-form/87958#87958
+
+Some Zulip discussion at https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/.F0.9D.94.BD.E2.82.82.5B.CE.B1.2C.20.CE.B2.2C.20.CE.B3.5D.20.2F.20.28.CE.B1.C2.B2.2C.20.CE.B2.C2.B2.2C.20.CE.B3.C2.B2.29/near/222716333.
+-/
+
+noncomputable section
+
+open scoped BigOperators
+
+namespace Q60596
+
+open MvPolynomial
+
+/-- The monomial ideal generated by terms of the form $x_ix_i$. -/
+def kIdeal : Ideal (MvPolynomial (Fin 3) (ZMod 2)) :=
+  Ideal.span (Set.range fun i => (X i * X i : MvPolynomial (Fin 3) (ZMod 2)))
+
+theorem mem_kIdeal_iff (x : MvPolynomial (Fin 3) (ZMod 2)) :
+    x ∈ kIdeal ↔ ∀ m : Fin 3 →₀ ℕ, m ∈ x.support → ∃ i, 2 ≤ m i := by
+  have :
+      kIdeal = Ideal.span ((monomial · (1 : ZMod 2)) '' Set.range (Finsupp.single · 2)) := by
+    simp_rw [kIdeal, X, monomial_mul, one_mul, ← Finsupp.single_add, ← Set.range_comp,
+      Function.comp]
+  rw [this, mem_ideal_span_monomial_image]
+  simp
+
+theorem X0_X1_X2_nmem_kIdeal : (X 0 * X 1 * X 2 : MvPolynomial (Fin 3) (ZMod 2)) ∉ kIdeal := by
```

Comment:
```suggestion
theorem X0_X1_X2_not_mem_kIdeal : (X 0 * X 1 * X 2 : MvPolynomial (Fin 3) (ZMod 2)) ∉ kIdeal := by
```
Just to follow standard mathlib conventions.

Reply:
We actually have a few `nmem`s in mathlib too, but indeed `not_mem` has the clear majority.